tests/child_helper.lua

```diff
@@ -105,6 +105,14 @@ function M.new_child_neovim(test_name)
     ]])
   end
 
+  function child.wait_for_lsp_authentication()
```

Comment:
@AntoineGS I was able to get the rest of the tests passing with this new helper that waits until `is_authenticated()` is true